### DISCUSSION TITLE
Strange Open → Browse behavior?

### DISCUSSION THREAD

**Discussion by Valour549** [WinMerge/winmerge#2583] [General]
After using Open → Browse, I try to select a folder for comparison, but every time I click Open it just opens the folder deeper.

How do I get it to actually select the folder?

**Comment by sdottaka** [comment]
The Open window that appears when you press the Browse button allows you to select both files and folders, so its operation differs from the standard Select Folder window.

After entering the folder you want to select, press the Open button without selecting anything.

**Comment by Valour549** [comment]
Thanks
